agent/config/config.go

```diff
@@ -520,6 +524,7 @@ func environmentConfig() (Config, error) {
 		TaskIAMRoleEnabledForNetworkHost:    utils.ParseBool(os.Getenv("ECS_ENABLE_TASK_IAM_ROLE_NETWORK_HOST"), false),
 		ImageCleanupDisabled:                utils.ParseBool(os.Getenv("ECS_DISABLE_IMAGE_CLEANUP"), false),
 		MinimumImageDeletionAge:             parseEnvVariableDuration("ECS_IMAGE_MINIMUM_CLEANUP_AGE"),
+		NonECSMinimumImageDeletionAge:       parseEnvVariableDuration("NON_ECS_IMAGE_MINIMUM_CLEANUP_WAIT_DURATION_SINCE_CREATED"),
```

Comment:
Can we just name it as **NON_ECS_IMAGE_MINIMUM_CLEANUP_AGE** since it serves the same purpose of **ECS_IMAGE_MINIMUM_CLEANUP_AGE**? And for more implementation details, we just keep it in README and code comments.